### PR TITLE
Ignore empty strings paths while calculting disk usage.

### PR DESCRIPTION
**Pull request by vishh**
cc @ncdc

### CODE REVIEW

**Comment by vishh** [comment]
@timstclair: This PR fixes the build.

File: container/docker/fsHandler.go [hunk 1]
```diff
@@ -67,15 +67,24 @@ func newFsHandler(period time.Duration, rootfs, extraDir string, fsInfo fs.FsInf
...
 	// TODO(vishh): Add support for external mounts.
-	baseUsage, err := fh.fsInfo.GetDirUsage(fh.rootfs, duTimeout)
-	if err != nil {
-		return err
+	if fh.rootfs != "" {
+		baseUsage, err = fh.fsInfo.GetDirUsage(fh.rootfs, duTimeout)
+		if err != nil {
+			return err
+		}
+		glog.Error(baseUsage)
```
Comment (ncdc) [inline container/docker/fsHandler.go]: I'm guessing this is something you put in for debugging that should be removed?

File: fs/fs.go [hunk 2]
```diff
@@ -358,6 +358,9 @@ func (self *RealFsInfo) GetDirFsDevice(dir string) (*DeviceInfo, error) {
 }
 
 func (self *RealFsInfo) GetDirUsage(dir string, timeout time.Duration) (uint64, error) {
+	if dir == "" {
+		return 0, nil
```
Comment (timstclair) [inline fs/fs.go]: Wouldn't it make more sense to return an error here?

File: container/docker/fsHandler.go (hunk 1 above)
Comment (vishh) [inline container/docker/fsHandler.go]: Yeah. Sorry. Removed it.

File: fs/fs.go (hunk 2 above)
Comment (vishh) [inline fs/fs.go]: Ok. Updated.

**Comment by vishh** [comment]
PTAL @timstclair

**Comment by timstclair** [comment]
LGTM

**Pull request merged**